docs/docs/typescript/introduction.md

```diff
@@ -132,3 +132,21 @@ import { CustomModal } from '@adminUI/CustomModal'
 1. **Improved code readability**, by abstracting complex directory hierarchies, and having meaningful names for your imports.
 1. **Code maintainability**, aliases allow you to decouple your code from the file structure and more easily move files around, as they are not tied to the longer path.
 1. **Reduce boilerplate**, no more `../../src/components/modules/admin/common/ui/` 😮‍💨
+
+When you start writing tests for components that contain alias paths, you will need to add the following to your Jest configuration in `jest.config.js`:
+
+```js
+const config = {
+  rootDir: '../',
+  preset: '@redwoodjs/testing/config/jest/web',
+  moduleNameMapper: {
+    '^@startRW/(.*)$': '<rootDir>/web/src/components/admin/$1',
```

Comment:
Q: Do you think it would be better if we matched the `adminUI` that is used as the example above?